tutorial/connect-from-aks/ConnectFromAKS/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Azure.Core;
+using Azure.Identity;
+using StackExchange.Redis;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace ConnectFromAKSSample
```

Comment:
This explicit namespace/class/Main stuff is the old style - consider using the new style (https://learn.microsoft.com/en-us/dotnet/core/tutorials/top-level-templates)